e2e/test/scenarios/admin/datamodel/editor.cy.spec.js

```diff
@@ -185,6 +287,10 @@ const setInputValue = (oldValue, newValue) => {
   cy.findByDisplayValue(oldValue).clear().type(newValue).blur();
 };
 
+const clearInputValue = oldValue => {
```

Comment:
function name is misleading as it also triggers "blur", can it be something like `clearAndBlurInput`?

Reply:
@uladzimirdev fixed, please take a look